src/main/resources/CollectStatistics/jsonStatisticsToCsv.xslt

```diff
@@ -1,64 +1,64 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:j="http://www.w3.org/2013/XSL/json">
-	<xsl:output method="text" />
+<xsl:output method="text" />
```

Comment:
De leesbaarheid van deze xslt is er niet op vooruit gegaan door de formattering te verwijderen.